src/main/java/au/com/dmg/fusion/request/paymentrequest/SaleItem.java

```diff
@@ -385,6 +391,25 @@ public Builder addTags(String tags) {
             return Builder.this;
         }
 
+        public Builder customFields(List<CustomField> customFields) {
+            if(customFields == null){
+                this.customFields = new ArrayList<>();
+            } else {
+                this.customFields = customFields;
+            }
+            return Builder.this;
+        }
+
+        public Builder addCustomField(CustomField customField) {
+            this.customFields.add(customField);
+            return Builder.this;
+        }
+
+        public Builder addCustomFields(List<CustomField> customFields) {
+            this.customFields.addAll(customFields);
```

Comment:
Please check if customFields (parameter value) is null before adding to this.customFields.